retro/retro_types.py

```diff
@@ -682,6 +682,7 @@ class FitStatus(enum.IntEnum):
     FailedToConverge = 30
     MissingSeed = 40
     InsufficientQuality = 50
+    Skipped = 60
```

Comment:
Note that I created my own arbitrary code `PositiveLLH = 1` since it seems all IceCube codes are divisible by 10. Unless we can force this into IceCube codebase, maybe someone comes up with their own code in parallel, and I'm betting it's going to be divisible by 10. So I'd recommend (a) make the code e.g. 2 and (b) make sure to add a comment as I did above that it is a custom code for Retro not present in IceCube/icetray software.

Reply:
ah, so the 10, 20, ... ones are I3 things? I'll change to 2 then

Reply:
That's correct. I tried to keep fidelity to I3 software wherever I could, so all the tuples and codes should be taken directly from there, unless otherwise noted. I tried to note in docstrings where I did get things from I3 software; in this case: `Scraped from dataclasses/public/dataclasses/physics/I3Particle.h, 2019-02-18`